website/docs/r/sm_private_certificate_configuration_action_set_signed.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_sm_private_certificate_configuration_action_set_signed"
+description: |-
+Manages PrivateCertificateConfigurationActionSetSigned.
+subcategory: "Secrets Manager"
+---
+
+# ibm_sm_private_certificate_configuration_action_set_signed
+
+Provides a resource for PrivateCertificateConfigurationActionSetSigned. This allows PrivateCertificateConfigurationActionSetSigned to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "ibm_sm_private_certificate_configuration_action_set_signed" "set_signed_action" {
+  instance_id           = "6ebc4224-e983-496a-8a54-f40a0bfa9175"
+  region                = "us-south"
+  name    = "my_configuration"
+  certificate                   = "-----BEGIN CERTIFICATE-----\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA\nAAAAAAAAAAAAAAAAAAAAAA==\n-----END CERTIFICATE-----"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your resource.
+
+* `instance_id` - (Required, Forces new resource, String) The GUID of the Secrets Manager instance.
+* `region` - (Required, Forces new resource, String) The region of the Secrets Manager instance. If not provided defaults to the region defined in the IBM provider configuration.
+* `endpoint_type` - (Optional, String) - The endpoint type. If not provided the endpoint type is determined by the `visibility` argument provided in the provider configuration.
+    * Constraints: Allowable values are: `private`, `public`.
+* `name` - (Required, Forces new resource, String) The name that uniquely identifies the configuration that will be used to sign the CSR.
+* `certificate` - (Required, Forces new resource, String) The PEM-encoded certificate.
+
+## Provider Configuration
```

Comment:
Following sections are redundant. It is generic and is available [here](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs#authentication). I understand that this is generated. I have informed generator team to fix this issue. Kindly remove following section from docs.
```
## Provider Configuration
## Static credentials
## Environment variables
```

Same comment applies to csr doc